### ISSUE TITLE
[BUG]: Recent Clappr Player update kills playback, console log gives " Uncaught TypeError: _clappr is undefined" for all the script i used

### ISSUE THREAD

**Issue by cometstyles** [clappr/clappr#2221]
### What version of Clappr are you using?

0.11.4/latest

### What browser (including version) are you using?

Firefox 115.18.0esr (64 bit)

### What OS (including version) are you using?

Windows 8

### Configuration

```JavaScript
<html>
<head>
<body style="background-color:black;"></body>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<script src="https://cdn.jsdelivr.net/npm/@clappr/player@latest/dist/clappr.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/level-selector@latest/dist/level-selector.min.js"></script>
<script src="https://cdn.jsdelivr.net/gh/clappr/dash-shaka-playback@latest/dist/dash-shaka-playback.js"></script>
<script src="https://cdn.jsdelivr.net/npm/clappr-chromecast-plugin@latest/dist/clappr-chromecast-plugin.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/clappr@latest/dist/clappr.min.js"></script>
<script defer src="https://www.gstatic.com/cv/js/sender/v1/cast_sender.js"></script>

</head>

<body>
<div id="player"></div>
<script>
var playerElement = document.getElementById("player-wrapper");
var player = new Clappr.Player({
  source: window.atob("stream link removed"),
  mimeType: 'application/vnd.apple.mpegurl',
  height: '100%',
  width: '100%',
  plugins: [DashShakaPlayback, LevelSelector],
  
  chromecast: {
    appId: '9DFB77C0',
    contentType: 'video/mp4',
  },
  shakaConfiguration: {
    drm: {
      clearKeys: {
        '':''
        // If you want to add more keys, separate them with a comma and follow the same format
        // 'KID': 'KEY',
        // 'KID': 'KEY',
        // 'KID': 'KEY'
      }
    },
    preferredAudioLanguage: 'en-EN',

  },
  shakaOnBeforeLoad: function(shaka_player) {},
  parentId: '#player'
});
</script>
</body>
</html>
```

### Steps to reproduce

1. Randomly started today (21st December) but definitely due to recent clappr update to 0.11.4, cause i replaced the player@latest in my scripts and changed it to previous version "<script src="https://cdn.jsdelivr.net/npm/@clappr/player@0.11.3/dist/clappr.min.js "></script>" and it started working again
2. My browser caches thus why it  took 2 days for me to realise something messed up after a hard reset of my browser's cache...please fix cause i get  these errors " Uncaught ReferenceError: ClapprCore is not defined/Uncaught TypeError: p is undefined/Uncaught TypeError: _clappr is undefined/ Uncaught ReferenceError: DashShakaPlayback is not defined etc.....all my script gets this undefined error

### Checklist

- [X] The issue occurs in the latest clappr main branch and not just on my page
- [X] The issue observed is not already reported by searching on Github under https://github.com/clappr/clappr/issues

**Comment by moszkowski97** [comment]
It was a bug! I was going crazy trying to resolve this issue.
It can clearly be seen that this has to be a bug in the 0.11.4 version as http://clappr.io/demo/ is not working.

**Comment by cometstyles** [comment]
> It was a bug! I was going crazy trying to resolve this issue. It can clearly be seen that this has to be a bug in the 0.11.4 version as http://clappr.io/demo/ is not working.

oh Thank god it wasn't just me, when i tested it on 2 other browsers and same outcome i know the issue wasn't me as this method has worked for me for 4 years now, i hope they notice this and fix it soon, or its gonna be a pain in the rear to try and fix this for 300 of these streams i created for personal use on phone and tv..

**Comment by cometstyles** [comment]
Yeah you people are useless, its christmas time and i can't watch my stuff, i'll switch to that other player...

**Comment by jhonatangcavalcanti** [comment]
0.11.5 fixes this problem
